ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Worker.java

```diff
@@ -364,15 +364,13 @@ protected Boolean findNextCompactionAndExecute(boolean collectGenericStats, bool
         t1 = resolveTable(ci);
         if (t1 == null) {
           LOG.info("Unable to find table " + ci.getFullTableName() +
-                       ", assuming it was dropped and moving on.");
-          msc.markCleaned(CompactionInfo.compactionInfoToStruct(ci));
```

Comment:
We have to move this compaction to completed_compactions somehow. Or am I missing something?

Reply:
According to @deniskuzZ in this case the underlying record already deleted from the table as a part of the DROP process. So there's no need to create a record in COMPLETED_COMPACTIONS.

Reply:
But if we're here then the table was just in the COMPACTION_QUEUE with state "initiated"... or maybe this whole block is unnecessary now, idk

Reply:
Oh my bad, I see what you mean